kubetest/main.go

```diff
@@ -734,7 +734,7 @@ func prepare(o *options) error {
 	}
 
 	switch o.provider {
-	case "gce", "gke", "kubemark", "kubernetes-anywhere", "node":
+	case "gce", "gke", "kubernetes-anywhere", "node":
```

Comment:
do we plan to get rid of `--provider=kubemark`? (from log dump)

Reply:
Yes.. I'll follow it up